news_update.php

```diff
@@ -69,17 +69,15 @@
 
 echo '<div class="space-20"></div>';
 echo '<div class="alert alert-success">';
-echo lang_get( 'operation_successful' ) ?>
-
-echo '<br />';
+echo lang_get( 'operation_successful' );
+echo '<br /></div>';
 
 print_button( 'news_edit_page.php?news_id=' . $f_news_id . '&action=edit', lang_get( 'edit_link' ) );
+echo '&nbsp;&nbsp;&nbsp;&nbsp;';
```

Comment:
As I understood, Mantishub LLC, represented here by @syncguru, purchased Extended License of   Ace Admin template (AAT) and incorporated it into the opensource MantisBT project, making AAT subject to GPL, which AAT's authors would not approve and sure might kindly ask to remove copyrighted materials from the MantisBT project.